OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -423,6 +433,10 @@ function ReceiveServerInfo(string sIP,coerce int iNumP,coerce int iMaxP,string s
 			else
 				CurServer.bSameVersion = true;
 			CurServer.bLocked = bSvrLocked;//1.5 added locked here
+			//1.5 add ping
+			i = Timer.EndTimer(CurServer.szIPAddr);

```

Comment:
For performance purposes, maybe move this above the iMaxPlayers setting? If this entire loop iteration happens in one tick then ignore me.

Reply:
Yep all in one tick.